### ISSUE TITLE
TypeError: Cannot convert object to primitive value

### ISSUE THREAD

**Issue by barhoumio** [Automattic/mongoose#6538]
**Do you want to request a *feature* or report a *bug*?**
Report a bug.

**What is the current behavior?**
When we override mongoose custom methods like toString() on a schema, then we console.log with a template literals expressions string, we got this error.
``` sh
 TypeError: Cannot convert object to primitive value
    at Comment.findOne.then.comment (/test.js:37:58)
```
**If the current behavior is a bug, please provide the steps to reproduce.**
repro script:
```js
const mongoose = require('mongoose');
const assert = require('assert');
mongoose.connect('mongodb://localhost:27017/test');
const connection = mongoose.connection;

mongoose.plugin(function (schema) {

  schema.methods.toString = function () {
    return `${this.constructor.modelName}(${this._id})`;
  };

});

let commentSchema = new mongoose.Schema({
  title: {
    type: String,
    index: true
  }
});

const Comment = mongoose.model('Comment', commentSchema);

async function run() {
  for (let i in [...Array(10).keys()]) {
    let comment = new Comment({
      title: `comment${i}`,
    });

    await comment.save();
  }

  await Comment
    .findOne({
      title: 'comment5'
    })
    .then(comment => {
        assert.strictEqual(`Comment(${comment._id})`, `${comment}`);
      }
    );

  return connection.close();
};

run().catch(console.error);
```
**What is the expected behavior?**
Override mongoose custom methods without using the createWrapper of Kareem lib if the method  does  not have a hook ( look at /mongoose/lib/services/model/applyHooks.js:67)
**Please mention your node.js, mongoose and MongoDB version.**
Node v9.11.1
MongoDB v3.2.20
Mongoose v5.1.3

**Comment by vkarpov15** [comment]
Thanks for reporting, will fix ASAP :+1:

**Comment by vkarpov15** [comment]
Fixed, fix will be in 5.1.5. Thanks for reporting with a standalone repro script, made it easy to fix :+1:

**Comment by barhoumio** [comment]
Cool 😃 thank you so much 🍾
